client/components/app.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'styled-components';
+import Chart5y from './chart5y.jsx';
```

Comment:
capitalize the `Chart5y.jsx` file name (as per Airbnb)

also, there's no need to name files `.jsx`; would advise `.js` instead